### PR TITLE
Exclude test_package/build from KB028

### LINKED ISSUE #117: [conan-center] hook should ignore test_package/build directory
currently, the hook reports many errors against generated CMake files:
```

[HOOK - conan-center.py] pre_export(): ERROR: [CMAKE MINIMUM VERSION (KB-H028)] The CMake file 'test_package\build\6e5a500f4408bf62c2f4c1c5c78987cea976e30d\CMakeCache.txt' must contain a minimum version declared (e.g. cmake_minimum_required(VERSION 3.1.2)) (https://github.com/conan-io/conan-center-index/wiki/Error-Knowledge-Base#KB-H028)
[HOOK - conan-center.py] pre_export(): ERROR: [CMAKE MINIMUM VERSION (KB-H028)] The CMake file 'test_package\build\6e5a500f4408bf62c2f4c1c5c78987cea976e30d\cmake_install.cmake' must contain a minimum version declared (e.g. cmake_minimum_required(VERSION 3.1.2)) (https://github.com/conan-io/conan-center-index/wiki/Error-Knowledge-Base#KB-H028) 
[HOOK - conan-center.py] pre_export(): ERROR: [CMAKE MINIMUM VERSION (KB-H028)] The CMake file 'test_package\build\6e5a500f4408bf62c2f4c1c5c78987cea976e30d\CMakeFiles\3.11.1\CMakeCCompiler.cmake' must contain a minimum version declared (e.g. cmake_minimum_required(VERSION 3.1.2)) (https://github.com/conan-io/conan-center-index/wiki/Error-Knowledge-Base#KB-H028)
[HOOK - conan-center.py] pre_export(): ERROR: [CMAKE MINIMUM VERSION (KB-H028)] The CMake file 'test_package\build\6e5a500f4408bf62c2f4c1c5c78987cea976e30d\CMakeFiles\3.11.1\CMakeCXXCompiler.cmake' must contain a minimum version declared (e.g. cmake_minimum_required(VERSION 3.1.2)) (https://github.com/conan-io/conan-center-index/wiki/Error-Knowledge-Base#KB-H028)
[HOOK - conan-center.py] pre_export(): ERROR: [CMAKE MINIMUM VERSION (KB-H028)] The CMake file 'test_package\build\6e5a500f4408bf62c2f4c1c5c78987cea976e30d\CMakeFiles\3.11.1\CMakeRCCompiler.cmake' must contain a minimum version declared (e.g. cmake_minimum_required(VERSION 3.1.2)) (https://github.com/conan-io/conan-center-index/wiki/Error-Knowledge-Base#KB-H028)
[HOOK - conan-center.py] pre_export(): ERROR: [CMAKE MINIMUM VERSION (KB-H028)] The CMake file 'test_package\build\6e5a500f4408bf62c2f4c1c5c78987cea976e30d\CMakeFiles\3.11.1\CMakeSystem.cmake' must contain a minimum version declared (e.g. cmake_minimum_required(VERSION 3.1.2)) (https://github.com/conan-io/conan-center-index/wiki/Error-Knowledge-Base#KB-H028)
[HOOK - conan-center.py] pre_export(): ERROR: [CMAKE MINIMUM VERSION (KB-H028)] The CMake file 'test_package\build\e6a9ba05783610684cd9c2bc7d38e8480168227f\CMakeCache.txt' must contain a minimum version declared (e.g. cmake_minimum_required(VERSION 3.1.2)) (https://github.com/conan-io/conan-center-index/wiki/Error-Knowledge-Base#KB-H028)
[HOOK - conan-center.py] pre_export(): ERROR: [CMAKE MINIMUM VERSION (KB-H028)] The CMake file 'test_package\build\e6a9ba05783610684cd9c2bc7d38e8480168227f\cmake_install.cmake' must contain a minimum version declared (e.g. cmake_minimum_required(VERSION 3.1.2)) (https://github.com/conan-io/conan-center-index/wiki/Error-Knowledge-Base#KB-H028) 
[HOOK - conan-center.py] pre_export(): ERROR: [CMAKE MINIMUM VERSION (KB-H028)] The CMake file 'test_package\build\e6a9ba05783610684cd9c2bc7d38e8480168227f\CMakeFiles\3.11.1\CMakeCCompiler.cmake' must contain a minimum version declared (e.g. cmake_minimum_required(VERSION 3.1.2)) (https://github.com/conan-io/conan-center-index/wiki/Error-Knowledge-Base#KB-H028)
[HOOK - conan-center.py] pre_export(): ERROR: [CMAKE MINIMUM VERSION (KB-H028)] The CMake file 'test_package\build\e6a9ba05783610684cd9c2bc7d38e8480168227f\CMakeFiles\3.11.1\CMakeCXXCompiler.cmake' must contain a minimum version declared (e.g. cmake_minimum_required(VERSION 3.1.2)) (https://github.com/conan-io/conan-center-index/wiki/Error-Knowledge-Base#KB-H028)
[HOOK - conan-center.py] pre_export(): ERROR: [CMAKE MINIMUM VERSION (KB-H028)] The CMake file 'test_package\build\e6a9ba05783610684cd9c2bc7d38e8480168227f\CMakeFiles\3.11.1\CMakeRCCompiler.cmake' must contain a minimum version declared (e.g. cmake_minimum_required(VERSION 3.1.2)) (https://github.com/conan-io/conan-center-index/wiki/Error-Knowledge-Base#KB-H028)
[HOOK - conan-center.py] pre_export(): ERROR: [CMAKE MINIMUM VERSION (KB-H028)] The CMake file 'test_package\build\e6a9ba05783610684cd9c2bc7d38e8480168227f\CMakeFiles\3.11.1\CMakeSystem.cmake' must contain a minimum version declared (e.g. cmake_minimum_required(VERSION 3.1.2)) (https://github.com/conan-io/conan-center-index/wiki/Error-Knowledge-Base#KB-H028)
```
/cc @uilianries

### PR DESCRIPTION
**Pull request by uilianries**
When a package is built for its first time, the cmake files generated on test_package/build folder will remain for the next build. However, the hook KB-H028 is scanning that folder which should not validate as part of recipe context.

fixes #117

### CODE REVIEW

File: tests/test_hooks/conan-center/test_conan-center.py [hunk 1]
```diff
@@ -397,3 +397,30 @@ def test_cmake_minimum_version(self):
...
+        """
+        tools.save('conanfile.py', content=conanfile)
+        tools.save('CMakeLists.txt', content=cmake)
+        tools.save('test_package/CMakeLists.txt', content=cmake)
+        tools.save('test_package/conanfile.py', content=conanfile_tp)
+        output = self.conan(['create', '.', 'name/version@user/test'])
+        self.assertIn("[CMAKE MINIMUM VERSION (KB-H028)] OK", output)
+        # validate residual cmake files in test_package/build
+        output = self.conan(['create', '.', 'name/version@user/test'])
+        self.assertIn("[CMAKE MINIMUM VERSION (KB-H028)] OK", output)
```
Comment (danimtb) [inline tests/test_hooks/conan-center/test_conan-center.py]: Maybe add an `assertNotIn()` with the message that this PR is avoiding?

Comment (uilianries) [reply tests/test_hooks/conan-center/test_conan-center.py]: do you mean "ERROR [CMAKE MINIMUM VERSION (KB-H028)] message" ?

Comment (danimtb) [reply tests/test_hooks/conan-center/test_conan-center.py]: yes

Comment (uilianries) [reply tests/test_hooks/conan-center/test_conan-center.py]: done

**Pull request merged**